src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java

```diff
@@ -177,21 +170,21 @@ else if(!Strings.isNullOrEmpty(injectedUserHeader)) {
 
             boolean skipSecurityIfDualMode = getThreadContext().getTransient(ConfigConstants.SECURITY_SSL_DUAL_MODE_SKIP_SECURITY) == Boolean.TRUE;
 
-                if(skipSecurityIfDualMode) {
-                    if(getThreadContext().getTransient(ConfigConstants.OPENDISTRO_SECURITY_REMOTE_ADDRESS) == null) {
-                        getThreadContext().putTransient(ConfigConstants.OPENDISTRO_SECURITY_REMOTE_ADDRESS, request.remoteAddress());
-                    }
+            if(skipSecurityIfDualMode) {
```

Comment:
fixed indentation to promote better readability, as it was misaligned in the file